sonic_sfp/sfputilbase.py

```diff
@@ -16,6 +16,7 @@
     from .sff8472 import sff8472Dom    # Dot module supports both Python 2 and Python 3 using explicit relative import methods
     from .sff8436 import sff8436InterfaceId  # Dot module supports both Python 2 and Python 3 using explicit relative import methods
     from .sff8436 import sff8436Dom    # Dot module supports both Python 2 and Python 3 using explicit relative import methods
+    from .sff8436 import sff8436DomThreshold # Dot module supports both Python 2 and Python 3 using explicit relative import methods
```

Comment:
To provide a complete solution, I would suggest implementing the same set of APIs also for  sff8472.

Reply:
Hi,
Implemented the same solution for sff8472

Reply:
@sridhar-ravindran if my understanding is correct, sff8436DomThreshold has already been removed from the code, so this line should be removed.